packages/rebber/src/types/table.js

```diff
@@ -0,0 +1,31 @@
+/* Expose. */
+const one = require('../one')
+module.exports = table
+
+const defaultMacro = (ctx, node) => {
+  let j = 0
+  const parsed = []
+  node.children.map(n => parsed.push(one(ctx, n, j++, node)))
```

Comment:
I think you're looking for `const parsed = node.children.map(n => one(ctx, n, j++, node))`